codereview2.java

```diff
@@ -4,7 +4,7 @@ public static void main(String[] args) {
         int[] numbers = {1, 2, 3};
         System.out.println(numbers[3]); 
 
-        
+        System.out.println(numbers[4]);
```

Comment:
<b>code: System.out.println(numbers[3]); - </b>
Array index out of bounds error, should be careful with index values